src/main/java/org/sagebionetworks/bridge/exporter/synapse/SynapseHelper.java

```diff
@@ -311,102 +312,6 @@ public static boolean isCompatibleColumn(ColumnModel oldColumn, ColumnModel newC
-        return createOrUpdateS3StorageLocationForProject(projectId, storageLocationSetting);
-    }
-
-    /**
-     * <p>
-     * Get the S3 storage location ID for the given project. Returns null if there is no S3 storage location.
-     * </p>
-     * <p>
-     * Package-scoped for unit tests.
-     * </p>
-     */
-    Long getS3StorageLocationIdForProject(String projectId) throws SynapseException {
-        UploadDestinationLocation[] locationArray = getUploadDestinationLocationsWithRetry(projectId);
-        for (UploadDestinationLocation location : locationArray) {
-            if (location.getUploadType() == UploadType.S3 &&
-                    location.getStorageLocationId() != DEFAULT_STORAGE_LOCATION_ID) {
-                return location.getStorageLocationId();
-            }
-        }
-
-        return null;
-    }
-
-    /**
-     * <p>
-     * Creates the storage location setting and adds it to the given project. Returns the storage location ID.
-     * </p>
-     * <p>
-     * Package-scoped for unit tests.
-     * </p>
-     */
-    long createOrUpdateS3StorageLocationForProject(String projectId, StorageLocationSetting storageLocationSetting)
-            throws SynapseException {
-        // First, create the storage location setting.
-        StorageLocationSetting createdStorageLocationSetting = createStorageLocationSettingWithRetry(
-                storageLocationSetting);
-        long storageLocationId = createdStorageLocationSetting.getStorageLocationId();
-
-        // Check to see if the project has existing storage settings.
-        UploadDestinationListSetting projectSetting;
-        try {
-            projectSetting = (UploadDestinationListSetting) getProjectSettingWithRetry(projectId,
-                    ProjectSettingsType.upload);
-        } catch (SynapseNotFoundException ex) {
-            // This is normal, if the setting hasn't been created yet.
-            projectSetting = null;
-        }
-
-        if (projectSetting != null) {
-            // Project setting already exists. Add it to the list of existing storage locations.
-            projectSetting.getLocations().add(storageLocationId);
-            updateProjectSettingWithRetry(projectSetting);
-        } else {
-            // Project setting does not exist. Create one. Ensure that we include both the default storage location
-            // (Synapse) and our S3 bucket so that users can still upload files just fine.
-            UploadDestinationListSetting uploadDestinationListSetting = new UploadDestinationListSetting();
-            uploadDestinationListSetting.setLocations(ImmutableList.of(DEFAULT_STORAGE_LOCATION_ID,
-                    storageLocationId));
-            uploadDestinationListSetting.setProjectId(projectId);
-            uploadDestinationListSetting.setSettingsType(ProjectSettingsType.upload);
-            createProjectSettingWithRetry(uploadDestinationListSetting);
-        }
-
-        return storageLocationId;
-    }
-
```

Comment:
So basically, all Bridge projects use the same storage location in config, they don't vary by project?

Reply:
Correct, all Bridge projects use the same storage location, because storage location is simply a mapping to an S3 bucket. (This is a bit of an over-simplification, but it's essentially true.)